src/django/contricleaner/lib/handlers/list_row_handler.py

```diff
@@ -1,16 +1,21 @@
 from __future__ import annotations
 from abc import ABC, abstractmethod
-from typing import List
+from typing import List, Dict
 
 from contricleaner.lib.dto.list_dto import ListDTO
+from contricleaner.lib.exceptions.handler_not_set_error \
+    import HandlerNotSetError
 
 
 class ListRowHandler(ABC):
-    _next: ListRowHandler
+    _next: ListRowHandler = None
 
     def set_next(self, next: ListRowHandler):
         self._next = next
 
     @abstractmethod
-    def handle(self, rows: List[dict]) -> ListDTO:
-        pass
+    def handle(self, rows: List[Dict]) -> ListDTO:
+        if self._next:
+            return self._next.handle(rows)
+
+        raise HandlerNotSetError("Next Handler wasn't set.")
```

Comment:
As I understand, after changes in the handle method it shouldn't be an abstract method anymore.

Reply:
It should be an abstract method since child classes need to define their own logic, while still being able to utilize the parent logic to call the next handler.